src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/IGitFileManager.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Xml;
+using Newtonsoft.Json.Linq;
+using NuGet.Versioning;
+
+namespace Microsoft.DotNet.DarcLib;
+public interface IGitFileManager
+{
+    Task AddDependencyAsync(DependencyDetail dependency, string repoUri, string branch);
```

Comment:
Much appreciated if you add xml comments to each method for completeness. Usually the comment in GitFileManager is fine to copy/paste

Reply:
Done - I copied them over.